asab/library/providers/filesystem.py

```diff
@@ -38,7 +38,7 @@ async def list(self, path, recursive=True):
 
 		# Remove library path from the beginning of file names
 		library_path_to_replace = "{}/".format(os.path.abspath(self.LibraryBaseDir))
-		files_names_list = [i.replace(library_path_to_replace, '') for i in file_names]
+		files_names_list = [i.replace(library_path_to_replace, '', 1) for i in file_names]
```

Comment:
is it better? I kind of like it...

Reply:
Not really - you need to do the replacement at the string start - otherwise it is a bit unpredictable.